spec/requests/admin/screened_ip_addresses_controller_spec.rb

```diff
@@ -32,6 +33,12 @@
       expect(response.status).to eq(200)
       result = response.parsed_body
       expect(result.length).to eq(1)
+
+      get "/admin/logs/screened_ip_addresses.json", params: { filter: "5.0.0.1" }
+
+      expect(response.status).to eq(200)
+      result = response.parsed_body
+      expect(result.length).to eq(1)
```

Comment:
Minor suggestion but testing for length is not sufficient in my opinion since a wrong result can be returned and this test will still pass. I noted that the existing assertions above are doing so but I would recommend changing them too.